examples/c/ex_col_store.c

```diff
@@ -0,0 +1,555 @@
+#include <test_util.h>
+
+#define N_DATA 100
+
+static const char *home;
+
+typedef struct {
+    char day[5];
+    uint16_t hour;
+    uint8_t temp;
+    uint8_t humidity;
+    uint16_t pressure;
+    uint8_t wind;
+    uint8_t feels_like_temp;
+    uint16_t loc_lat;
+    uint16_t loc_long;
+    char country[5];
+} WEATHER;
```

Comment:
I am not sure if we have a convention around anonymous structures. We usually do typedef __weather WEATHER; I think it is okay to use anonymous structs.

Reply:
The other example (ex_schema) does it the same way, so I think it should be okay.